Tasks/DownloadGitHubReleasesV0/README.md

```diff
@@ -0,0 +1,31 @@
+# Download GitHub Releases
```

Comment:
Download GitHub Release?

Reply:
The task itself is called as Download GitHub Releases. We have to be consistent.

Reply:
renamed as suggested